docs/index.md

```diff
@@ -0,0 +1,260 @@
+
+## Identifying places nearby
+
+### Device location
+
+The nearby-app uses a [mapless app pattern](https://developers.arcgis.com/android/guide/determine-your-app-map-pattern.htm#ESRI_SECTION1_58C46384E3484890A47629F8F12E6EF5) by first presenting a list of nearby places.  Since the app starts with a list, rather than a map, the device location is obtained using Google’s Location Services API. In the future, the Runtime SDK can be used to obtain the device location outside of the `MapView`.  Before trying to obtain the device location, the app checks that the device's GPS and wireless settings are turned on and then configures Google's location service.
+
+```java
+// Google's location services are configured in the
+// PlacesFragment onCreate method.
+if (mGoogleApiClient == null) {
+    // Create an instance of GoogleAPIClient.
+    mGoogleApiClient = new GoogleApiClient.Builder(getActivity())
+      .addConnectionCallbacks(this)
+      .addOnConnectionFailedListener(this)
+      .addApi(LocationServices.API)
+      .build();
+}
+// Once Google's location service is connected,
+// you can use the device location to start the
+// geocoding search.
+@Override public void onConnected(@Nullable Bundle bundle) {
+    mLastLocation = LocationServices.FusedLocationApi.getLastLocation(mGoogleApiClient);
+    startSearch(mLastLocation);
+}
+```
+
+Once a location is obtained, a search for places can begin.
+
+### Geocoding
+
+The search for places uses category filters (e.g. “Hotel”, “Food”, “Pizza”) to find places matching these types near the current device location. The World Geocoding service uses a hierarchical structure of categories allowing high level concepts, like a category level 1 descriptor (e.g. “POI” for place of interest) to be searched as well as more specific category level 3 types like “Brazilian Food” or “Science Museum” to be used. The category filters and other search criteria are defined using the SDK’s [Geocode Parameters](https://developers.arcgis.com/android/latest/api-reference/reference/com/esri/arcgisruntime/tasks/geocode/GeocodeParameters.html).
+
+The geocode parameters are configured with the maximum number of results to return, the desired categories, the device's current location, and [output fields](https://developers.arcgis.com/rest/geocode/api-reference/geocoding-service-output.htm#ESRI_SECTION1_42D7D3D0231241E9B656C01438209440) using the following pattern.  Note that the geocoding service supports a specific list of categories defined [here](https://developers.arcgis.com/rest/geocode/api-reference/geocoding-category-filtering.htm#ESRI_SECTION1_502B3FE2028145D7B189C25B1A00E17B).  In the nearby-app, we're interested in the top 10 results within a radius of 50 kilometers of the current location.
```

Comment:
Following on the first comment, I recommend:
"the nearby-app" --> "Nearby Places"